sentinel-extension/sentinel-prometheus-exporter/src/main/java/com/alibaba/csp/sentinel/prom/PromExporterInit.java

```diff
@@ -0,0 +1,43 @@
+package com.alibaba.csp.sentinel.prom;
```

Comment:
Could you please prepend the license header here?

Reply:
has changed.